### ISSUE TITLE
erro ao enviar anexos

### ISSUE THREAD

**Issue by rodrigo-ogioni** [salvadordf/CEF4Delphi#480]
Boa tarde!!
em alguns clientes, no envio de mensagem, o anexo não é enviado. Em outros clientes vai certinho. O que pode estar acontecendo?
Grato

**Comment by salvadordf** [comment]
Hi,

Please, use [our forum](https://www.briskbard.com/forum/) for questions and use English or Spanish.

It's not possible to know what's the problem without some code to reproduce this issue in my computer.
Build the CEF binaries with proprietary codecs to avoid issues handling videos or audio clips using those codecs.

**Comment by rodrigo-ogioni** [comment]
> Hi,
> 
> Please, use [our forum](https://www.briskbard.com/forum/) for questions and use English or Spanish.
> 
> It's not possible to know what's the problem without some code to reproduce this issue in my computer. Build the CEF binaries with proprietary codecs to avoid issues handling videos or audio clips using those codecs.

good morning,
I pulled the files from a client that was working correctly. I saw that the js.abr file was different. So I put the same file in the one that wasn't working. However, it didn't work. I copied the entire folder and pasted it into the client and it worked normally with the new folder. I saw that the folder I copied to had several dcus. So I was unsure which files were needed for tinject to work. I compared the two folders, and the main files, dlls, are all the same, except the dcus.
Grateful
